cupyx/scipy/spatial/distance.py

```diff
@@ -628,7 +628,7 @@ def cdist(XA, XB, metric='euclidean', out=None, **kwargs):
             output_arr = out if out is not None else cupy.zeros((mA, mB),
                                                                 dtype=XA.dtype,
                                                                 order=XA_order)
-            pairwise_distance(XA, XB, output_arr, metric, p=p)
+            pairwise_distance(XA, XB, output_arr, metric, p)
```

Comment:
The simplest solution is just to pass this argument positionally. That appears to work across versions

Reply:
Yeah, if this is allowed then it's definitely the path of least resistence